src/Controls/tests/Xaml.Benchmarks/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using BenchmarkDotNet.Attributes;
+
+namespace Microsoft.Maui.Controls.Xaml.Benchmarks;
+
+[MemoryDiagnoser(false)]
+public class LayoutBenchmark
+{
+	[Benchmark(Baseline = true)]
+	public void XamlC()
+	{
+		var page = new UnitTests.Benchmark(XamlInflator.XamlC);
+	}
+
+	[Benchmark]
+	public void SourceGen()
+	{
+		var page = new UnitTests.Benchmark(XamlInflator.SourceGen);
+	}
+	[Benchmark]
+	public void Runtime()
+	{
+		var page = new UnitTests.Benchmark(XamlInflator.Runtime);
+	}
+}
+
+class Program
+{
+	static void Main(string[] args)
+	{		
+		BenchmarkDotNet.Running.BenchmarkRunner.Run<LayoutBenchmark>();
```

Comment:
If you want to have future Benchmarks in here, you can put:
```csharp
BenchmarkSwitcher.FromAssembly (typeof (Program).Assembly).Run (args);
```

It will prompt to select which one.